<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1131</width>
      <height>721</height>
     </rect>
    </property>
    <property name="title">
     <string>Stock</string>
    </property>
    <widget class="QTabWidget" name="tab_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1101</width>
       <height>681</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>produit </string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1071</width>
         <height>631</height>
        </rect>
       </property>
       <property name="title">
        <string>Ajouter un produit</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>441</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>marque      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="marque"/>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>type       </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="type"/>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>id produuit  </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_id"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>nom produit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_nom"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Reference</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_reference"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>QTE        </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="qte"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>80</y>
          <width>179</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>170</y>
          <width>179</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifer</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>400</y>
          <width>651</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>Liste des produits</string>
        </property>
        <widget class="QTableView" name="tableView">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>611</width>
           <height>191</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_10">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>250</y>
          <width>291</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>supprimer un produit</string>
        </property>
        <widget class="QPushButton" name="pushButton_supprimer">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_supprimer_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>260</y>
          <width>271</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Recherche</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>30</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>chercher</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>520</y>
          <width>381</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Suppression</string>
        </property>
        <widget class="QLabel" name="label_43">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>69</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="suppProduit">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>20</y>
          <width>221</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/logo/88.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>410</y>
          <width>381</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Tri</string>
        </property>
        <widget class="QRadioButton" name="radioButton_TRInProd">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>TRI nom</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_TRIqProd">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>20</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>TRI id produit</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>consulter stock</string>
      </attribute>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; text-decoration: underline; color:#00aa00;&quot;&gt;En stock&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>310</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; text-decoration: underline; color:#aa0000;&quot;&gt;TRI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>330</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>dec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cro</string>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>280</y>
         <width>211</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Recherche</string>
       </property>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>chercher</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>supprimer un produit</string>
       </property>
       <widget class="QPushButton" name="sss">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_supprimer_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>831</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string>Liste des produits</string>
       </property>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>801</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>passer commande</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>login</string>
      </attribute>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>240</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>50</y>
         <width>221</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/logo/88.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>180</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>cnx</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1176</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
